test/svm/SvmSpoke.Bundle.ts

```diff
@@ -9,26 +15,19 @@ import {
   TransactionMessage,
   VersionedTransaction,
 } from "@solana/web3.js";
+import { MerkleTree } from "@uma/common/dist/MerkleTree";
```

Comment:
nit: better import from `utils/MerkleTree.ts`. Though under the hood its the same uma common package import, it might be better to import it consistently from one source